.github/workflows/CI.yml

```diff
@@ -0,0 +1,33 @@
+# This workflow uses actions that are not certified by GitHub.
+# They are provided by a third-party and are governed by
+# separate terms of service, privacy policy, and support
+# documentation.
+# This workflow will build a Java project with Gradle and cache/restore any dependencies to improve the workflow execution time
+# For more information see: https://help.github.com/actions/language-and-framework-guides/building-and-testing-java-with-gradle
+
+name: Java CI with Gradle
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up JDK 11
+      uses: actions/setup-java@v2
+      with:
+        java-version: '11'
+        distribution: 'temurin'
+    - name: Grant execute permission for gradlew
```

Comment:
afaik not needed

Reply:
it is needed, without this grant the build is failing - https://github.com/tfij/MeterRegistryHolder/runs/5190241574?check_suite_focus=true
or I have to change file permisions and commit it. However, I prefer current way :)

Reply:
you can just use `run: ./gradlew build` instead of `gradle/gradle-build-action@937999e9cc2425eddc7fd62d1053baf041147db7`

Reply:
by doing so you don't need to 1. change file permissions in any way 2. you don't use another action, just run command

Reply:
```
/home/runner/work/_temp/92c858cf-bee1-44ca-bf90-914dcbc3fbe7.sh: line 1: ./gradlew: Permission denied
Error: Process completed with exit code 126.
```
https://github.com/tfij/MeterRegistryHolder/runs/5197230170?check_suite_focus=true

Reply:
you're right. I wonder how it started working in axion then, maybe sb called `git update-index --chmod=+x` on gradlew

Reply:
I guess so